pkg/local_workflows/auth_workflow.go

```diff
@@ -108,9 +110,11 @@ func entryPointDI(config configuration.Configuration, logger *zerolog.Logger, en
 
 		fmt.Println(auth.AUTHENTICATED_MESSAGE)
 	} else { // LEGACY flow
+		config.Unset(auth.CONFIG_KEY_OAUTH_TOKEN)
```

Comment:
**suggestion (non-blocking):** Consider introducing some logging that details the why behind the configuration changes. This may help folks in the future when debugging "unexpected" configuration changes around this.

Reply:
Good idea, did you have debug logging in mind?

Reply:
Updated, PTAL